src/EditorFeatures/VisualBasicTest/RemoveUnusedVariable/RemoveUnusedVariableTest.vb

```diff
@@ -73,6 +73,225 @@ Module M
     Sub Main()
     End Sub
 End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
+        Public Async Function RemoveUnusedVariableAndComment() As Task
+            Dim markup =
+<File>
+Module M
+    Sub Main()
+        Dim [|a|] As Integer ' inline comment also to be deleted. 
+    End Sub
+End Module
+</File>
+            Dim expected =
+<File>
+Module M
+    Sub Main()
+    End Sub
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/24076"), Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
```

Comment:
To workaround https://github.com/dotnet/roslyn/issues/24076, should we consider porting [BasicRemoveUnusedLocalsAnalyzer](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeQuality.Analyzers/VisualBasic/Maintainability/BasicRemoveUnusedLocals.vb) here and change this fixer to handle both `BC42024` and new IDE diagnostic generated by `BasicRemoveUnusedLocalsAnalyzer`?

Reply:
Will do. Thank you!

---
In reply to: [161331772](https://github.com/dotnet/roslyn/pull/24077#discussion_r161331772) [](ancestors = 161331772)